### DISCUSSION TITLE
Episode 209: Scrabble, Cribbage & More! (Board Games Part 2)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#108] [Episode]
Episode Link: https://adspthepodcast.com/2024/11/22/Episode-209.html

Feel free to write comments or ask questions 🙂

**Comment by bartoszluka** [comment]
Hey Connor, the "Tony's talk" got me interested but it's not linked anywhere. Do you have it? Or at least the name of the talk?

**Reply by codereport** [reply]
Ah thanks for catching that! I meant to add them but must have missed it while editing. I just added them but to save you the trip, if you search "Value Oriented Programming" (for me at least) on YouTube, 7 of the top 8 results are Tony talks. He skips around with the numbering just for fun (so don't be confused by that because II. III and IV are missing). Here are some of them:

* [Value Oriented Programming Part 1: You Say You Want To Write A Function - Tony Van Eerd CppNorth 22](https://www.youtube.com/watch?v=rpCc-cfYa3k&pp=ygUadmFsdWUgb3JpZW50ZWQgcHJvZ3JhbW1pbmc%3D)
* [Value Oriented Programming Part 1: You Say You Want to Write a Function - Tony Van Eerd CppNow 2023](https://www.youtube.com/watch?v=b4p_tcLYDV0)
* [Value Oriented Programming Part V: Return of the Values - Tony Van Eerd - CppNorth 2023](https://www.youtube.com/watch?v=gqopX4VInQw&pp=ygUadmFsdWUgb3JpZW50ZWQgcHJvZ3JhbW1pbmc%3D)
* [Value Oriented Programming Part V - Return of the Values - Tony Van Eerd - C++Now 2024](https://www.youtube.com/watch?v=sc1guyo5Rso&pp=ygUadmFsdWUgb3JpZW50ZWQgcHJvZ3JhbW1pbmc%3D)
